templates/github/.github/workflows/update-labels.yml.j2

```diff
@@ -0,0 +1,33 @@
+{% include 'header.j2' %}
+{% from 'macros.j2' import
+  checkout,
+  setup_python,
+  install_python_deps,
+  configure_git,
+with context %}
+
+---
+name: Update Labels
+on:
+  push:
+    paths:
+    - './template_config.yml'
+
+jobs:
+  update_backport_labels:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup Python
+      run: |
+        {{ setup_python() }}
+        {{ install_python_deps() }}
```

Comment:
Presumably this is a bit much.
All we need is requests. The rest is python stdlib.